src/com/dtmilano/android/viewclient.py

```diff
@@ -1124,8 +1124,10 @@ def flingBackward(self):
             raise RuntimeError('Not implemented yet')
         print >> sys.stderr, "flingBackward: view=", self.view.__smallStr__(), self.view.getPositionAndSize()
         print >> sys.stderr, "self.view.device.drag(%s, %s, %s, %s)" % (s, e, self.duration, self.steps)
-        self.view.device.drag(s, e, self.duration, self.steps, 0)
-    
+        if self.view.device.display['orientation'] == 0:
+            self.view.device.drag(s, e, self.duration, self.steps, 0)
```

Comment:
I think you can avoid the if by doing:

`self.view.device.drag(s, e, self.duration, self.steps, self.view.device.display['orientation'])`